app/views/admin/home_settings.html.erb

```diff
@@ -1,27 +1,56 @@
+<style>
+  .cards {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: stretch;
+  }
+  .card {
+    flex: 0 0 250px;
+    margin: 10px;
+  } 
+  .card .panel-heading a{
+    float: right;
+  }
+  .card img {
+    max-width: 100%;
+  }
+  .card .text {
+    padding: 0 20px 20px;
+  }
+  .card .text > button {
+    background: gray;
+    border: 0;
+    color: white;
+    padding: 10px;
+    width: 100%;
+    }
+</style>
```

Comment:
Can these be moved to a stylesheet rather than within the page?

Reply:
I've left these here for the time being. I unsure what the wider impact would be for moving them